README.md

```diff
@@ -108,25 +160,31 @@ The Dynatrace API token to be used by the exporter is specified using `setApiTok
 The token could, for example, be read from an environment variable.
 It should not be hardcoded into the code, especially if that code is stored in a VCS.
 
-Creating an API token for your Dynatrace environment is described in the [Dynatrace API documentation](https://www.dynatrace.com/support/help/dynatrace-api/basics/dynatrace-api-authentication/).
-The permission required for sending metrics is the `Ingest metrics` (`metrics.ingest`) permission in the **API v2** section
+Creating an API token for your Dynatrace environment is described in the
+[Dynatrace API documentation](https://www.dynatrace.com/support/help/dynatrace-api/basics/dynatrace-api-authentication/).
+The permission required for sending metrics is the `Ingest metrics` (`metrics.ingest`) permission in the **API v2** section,
 and it is recommended to limit scope to only this permission:
 
 ![API token creation](docs/img/api_token.png)
 
 #### Metric Key Prefix
 
-The `setPrefix` method of the builder specifies an optional prefix, which is prepended to each metric key, separated by a dot (e.g. a prefix of `<prefix>` and a metric name of `<name>` will lead to a combined metric name of `<prefix>.<name>`).
+The `setPrefix` method of the builder specifies an optional prefix, which is prepended to each metric key,
+separated by a dot (e.g. a prefix of `<prefix>` and a metric name of `<name>`
+will lead to a combined metric name of `<prefix>.<name>`).
 
 In the example, a prefix of `otel.java` is used, which leads to metrics named `otel.java.metric_name`, and allows for clear distinction between metrics from different sources in the Dynatrace metrics UI.
 
 #### Default Dimensions
 
-The `setDefaultDimensions` method can be used to optionally specify a `Labels` object, which will be added as additional dimensions to all data points.
-`Labels` represent key-value pairs.
-Dimension keys will be normalized, de-duplicated, and only one dimension value per key will be sent to the server.
+The `setDefaultDimensions` method can be used to optionally specify a
+[Attributes](https://github.com/open-telemetry/opentelemetry-java/blob/main/api/all/src/main/java/io/opentelemetry/api/common/Attributes.java) object,
+which will be added as additional dimensions to all data points. The `Attributes` interface represents key-value pairs.
+
+Dimension keys will be normalized, prior to be sent to the server.
```

Comment:
They are already de-duped by the SDK when using the Attributes built-in impl. So we don't need to mention this?

Reply:
I'd still mention it, since we combine default dimensions, static dimensions and the dimensions from the attributes. This might lead to people wondering why attributes were dropped, and having this line here makes it clear IMO